CleanCodeExercise.js

```diff
@@ -0,0 +1,70 @@
+// “Always code as if the guy who ends up maintaining your code will be a violent psychopath who knows where you live”
+// 
+// THIS IS A JAVASCRIPT-LIKE CODE IMPLEMENTING THE SCHEDULING OF BATHS AND VETERINARIANS ON THE PET STORE CASE.
+// 
+// PLEASE REVIEW THIS CODE TO GET IT AS CLEAN AS POSSIBLE. JUSTIFY YOUR CHANGES.
+// 
+
+
+var c;
+
+public class Costumer {
+
+	constructor(name, address, petName1, petName2, petName3, petName4, phoneNumber) {
+		this.name = height;
+		this.address = address;
+		this.petNames = [petName1, petName2, petName3, petName4];
+		this.phoneNumber = phoneNumber;
+	}
+	public getCustomer(id) {
+		return $http.get('myServer.com/getCustomer?id=' + id);
+	}
+	static all() {
+		return $http.get('myServer.com/returnAllCustomers');
+	}
+};
+
+public class Schedule {
+	constructor(dia, hora, duracao, customerId, type, serviceDescription, otherInfo) {
```

Comment:
Ou remover o serviceDescription do construtor ou inserir a abaixo da linha #34 o seguinte:
"this.serviceDescription = serviceDescription;"